<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonControls</class>
 <widget class="QWidget" name="ButtonControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pB_add">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/plus-16.png</normaloff>:/icons/plus-16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_remove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/delete-16.png</normaloff>:/icons/delete-16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_up">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Up</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/arrow_down-16.png</normaloff>:/icons/arrow_down-16.png</iconset>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pB_down">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Down</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/arrow-top-16.png</normaloff>:/icons/arrow-top-16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
